### ISSUE TITLE
Cloud Formation template missing BuildToolsAccountId

### ISSUE THREAD

**Issue by cmangla** [awslabs/aws-batch-cbmc#40]
On running `make install`, when it gets to the Cloud Formation template, I get the following error:

```
An error occurred (ValidationError) when calling the CreateStack operation: Parameters: [BuildToolsAccountId] must have values
```

I last used aws-batch-cbmc in November but didn't have this issue.

**Comment by cmangla** [comment]
@eidelmanjonathan I see that BuildToolsAccountId was added just after I finished my AWS Internship. Do I need to set this somewhere, like in an environment variable?

**Comment by eidelmanjonathan** [comment]
Hi @cmangla ! There have been a ton of changes in the last few months and it is very possible we have broken the Makefile. What exactly are you trying to do?

**Comment by cmangla** [comment]
Hi @eidelmanjonathan , for now, I'm just trying to build aws-batch-cbmc . I've setup `Makefile.local` with my AWS details and `make` worked fine. When I try `make install`, everything goes through fine, except when it gets to building the `template` directory, it complains with:
```
An error occurred (ValidationError) when calling the CreateStack operation: Parameters: [BuildToolsAccountId] must have values
```

A few months ago I didn't encounter this problem. I saw that `BuildToolsAccountId` was added in PR #12 to `cbmc.yaml`, so I thought that might be related.

My eventual goal is to run the CBMC proofs in aws-c-common, which uses aws-batch-cbmc.
